code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -125,17 +125,18 @@
 	description = "A chemical mixture with almost magical healing powers. Its main limitation is that the patient's body temperature must be under 270K for it to metabolise correctly."
 	color = "#0000C8"
 	taste_description = "blue"
+	creation_purity = REAGENT_STANDARD_PURITY
 	ph = 11
 	burning_temperature = 20 //cold burning
 	burning_volume = 0.1
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
 /datum/reagent/medicine/cryoxadone/on_mob_life(mob/living/carbon/affected_mob, delta_time, times_fired)
 	metabolization_rate = REAGENTS_METABOLISM * (0.00001 * (affected_mob.bodytemperature ** 2) + 0.5)
-	if(affected_mob.bodytemperature >= T0C || !HAS_TRAIT(affected_mob, TRAIT_KNOCKEDOUT))
+	if(affected_mob.bodytemperature >= T0C)
 		..()
 		return
-	var/power = -0.00003 * (affected_mob.bodytemperature ** 2) + 3
+	var/power = (-0.00004 * (affected_mob.bodytemperature ** 2) + 3) * normalise_creation_purity() * HAS_TRAIT(affected_mob, TRAIT_KNOCKEDOUT) ? 1 : 0.75
```

Comment:
Changing -0.00003 to -0.00004 actually makes it slightly slower. It increases the rate which temperature reduces the healing rate.

Reply:
Well ain't that nice to know.

Reply:
I've changed it back to how it was, it should no longer be slower.